app/styles/mobile.scss

```diff
@@ -87,36 +107,53 @@
 
 // Medium Devices, Desktops
 @media only screen and (min-width : 992px) {
+
 .btn-danger {
-    width:130px;
-    height: 55px;
-    margin-left: 5px;
-    margin-right: 5px;
-    margin-top: 18px;
-    margin-bottom: 18px;
+  height: 55px;
+  margin-bottom: 18px;
+  margin-left: 5px;
+  margin-right: 5px;
+  margin-top: 18px;
+  width: 130px;
   }
 
-#logo{
-    width: 200px;
-    height: 95px;
+#logo {
```

Comment:
Avoid using id selectors<br>Line should be indented 2 spaces, but was indented 0 spaces